packages/web3-providers/src/coingecko/index.ts

```diff
@@ -53,6 +53,8 @@ export class CoinGeckoAPI implements PriceAPI.Provider, TrendingAPI.Provider<Cha
 
     async getCoinTrending(chainId: ChainId, id: string, currency: TrendingAPI.Currency): Promise<TrendingAPI.Trending> {
         const info = await getCoinInfo(id)
+        if ('error' in info) throw new Error(info.error)
```

Comment:
https://coingecko-agent.r2d2.to/api/v3/coins/bored-ape-yacht-club?developer_data=false&community_data=false&tickers=true